templates/patterns/offcanvas/pattern-offcanvas.html.twig

```diff
@@ -16,7 +20,7 @@
   'extra_classes_close': extra_classes_close,
 } %}
 {% if with_trigger %}
-  <button class="btn btn-primary" type="button" data-bs-toggle="offcanvas" data-bs-target="#{{ offcanvas_id }}" aria-controls="{{ offcanvas_id }}">
+  <button class="btn btn-primary d-lg-none" type="button" data-bs-toggle="offcanvas" data-bs-target="#{{ offcanvas_id }}" aria-controls="{{ offcanvas_id }}">
     Toggle offcanvas
```

Comment:
Change the pattern, add a field for the button label, default to "Toggle offcanvas"